plugins/devo/help.md

```diff
@@ -0,0 +1,160 @@
+# Description
+
+[Devo](https://www.devo.com/) is the cloud-native logging and security analytics solution that delivers real-time visibility for security and operations teams
+
+# Key Features
+
+* Query for log results
+* Trigger workflows on new alerts
+
+# Requirements
+
+* A Devo authentication token
+
+The Devo authentication token must have access to the tables you are building your query for. In additon
+if you are using the alert trigger, the access token will need access to 
+`siem.logtrust.alert.info`
+
+For testing purposes, to give access to all tables, use `*.*.**`. This is not recommended for final production use, but can be used to rule out access errors when configuring the plugin for hte first time.  
+
+Authentication Token Documentation: 
+https://docs.devo.com/confluence/ndt/latest/domain-administration/security-credentials/authentication-tokens
```

Comment:
This should be a markdown link.

Reply:
It's intentionally not. I wasn't sure how the extensions library handled that.